### DISCUSSION TITLE
Hola buenos dias!!!! Busco 5 integrantes con nivel inicial en html, css, y Javascript para la hackathon

### DISCUSSION THREAD

**Discussion by Hernanjas400** [Launch-X-Latam/MisionFrontEnd#568] [Team Posts]
**Para quienes no saben que es una hackathon les dejo una breve reseña del evento** 

Un hackathon ​ designa, en la terminología usada en las comunidades hacker, un encuentro de programadores cuyo objetivo es el desarrollo colaborativo de software, aunque en ocasiones puede haber también un componente de hardware. Este tipo de encuentros puede durar entre dos días y una semana

En sus inicios comenzó siendo un espacio exclusivo para programadores, ya que el resultado era netamente tecnológico, pero hoy en día las hackathones también sirven para intercambiar ideas, compartir aprendizajes y colaborar en proyectos de otros usuarios

Formemos grupo juntos "Nivel inicial" : 

Mi correo es: lx06198@innovaccion.mx

Saludos!!!!!!

**Comment by IsraelVera** [comment]
Te mandé me sake por teams

Obtener Outlook para Android<https://aka.ms/AAb9ysg>

________________________________
De: Hernanjas400 ***@***.***>
Enviado: miércoles, 16 de noviembre de 2022 5:51 a. m.
Cc: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-backend] Hola buenos dias!!!! Busco 5 integrantes con nivel inicial en html, css, y Javascript para la hackathon (#105)


Para quienes no saben que es una hackathon les dejo una breve reseña del evento

Un hackathon ​ designa, en la terminología usada en las comunidades hacker, un encuentro de programadores cuyo objetivo es el desarrollo colaborativo de software, aunque en ocasiones puede haber también un componente de hardware. Este tipo de encuentros puede durar entre dos días y una semana

En sus inicios comenzó siendo un espacio exclusivo para programadores, ya que el resultado era netamente tecnológico, pero hoy en día las hackathones también sirven para intercambiar ideas, compartir aprendizajes y colaborar en proyectos de otros usuarios

Formemos grupo juntos "Nivel inicial" :

Mi correo es: ***@***.******@***.***>

Saludos!!!!!!

**Comment by Hernanjas400** [comment]
Hola Isray ya estas Incluido cuando completamos el cupo le ponemos un nombre y nos organizamos para presentarnos Saludos!!!!!!

**Comment by jacob-guerrero** [comment]
Me gustaría poder ingresar si aún queda espacio.
Mi correo es: lx11640@innovaccion.mx

**Comment by tdgarza** [comment]
Buen día. Si hay cupo lx16555@innovaccion.mx dejo mi correo.

**Comment by mixtlistar00** [comment]
Buen día me gustaría unirme a este equipo si aun hay espacio 

Mi correo es lx02113@innovaccion.mx 
Muchas gracias

**Comment by Deijux** [comment]
Buen día, dejo mi correo por si todavia hay cupo! 
lx07899@innovaccion.mx

**Comment by Hernanjas400** [comment]
Hola como va te mede mensaje por teams Saludos


Hernan Costanzi

Tel : 291 416-7016

________________________________
De: David ***@***.***>
Enviado: miércoles, 16 de noviembre de 2022 09:28
Cc: Hernanjas400 ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Hola buenos dias!!!! Busco 5 integrantes con nivel inicial en html, css, y Javascript para la hackathon (#105)


Buen día, dejo micorreo por si todavia hay cupo!
***@***.******@***.***>

**Comment by Hernanjas400** [comment]
Hola Chicos a todos les mande mensaje por teams confirmen por favor completamos Saludos

**Comment by Ayuya2001** [comment]
Buscaba esto la verdad, supongo el equipo ya está completo, si alguien más
vee esto y es nuevo hagamos equipo, yo aparte de este curso, comencé otro 2
meses antes hací que podría ir un poquito por delante de otro que comenzó
en este curso y es nuevo.

**Comment by Angelloinnovacion** [comment]
Yo también busco grupo

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Ayuya2001** [comment]
Hagamos grupo, te gustaría hacerlo tu o lo hago yo, yo la verdad no me
manejo con teams pero lo puedo intentar hacer

**Comment by Angelloinnovacion** [comment]
Puedes crearlo tu bro

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Ayuya2001** [comment]
Ok lo hago, me mandas tu correo de innovación, si lo mandaste no se vee.

**Comment by Angelloinnovacion** [comment]
***@***.***

Es ese

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Ayuya2001** [comment]
Amigo no se que pasa, tu correo aparece así *********
Tu puedes ver mi correo es este: ***@***.***
Si lo vees bien mándame un correo a este que es mi correo personal

Si no se vee no sé que podemos hacer

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Angelloinnovacion** [comment]
Yo tampoco puedo ver tu correo amigo
Q tal ahora?
lx06752 @ innovaccion.mx
Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Ayuya2001** [comment]
Ahora sí se vee, creo que ya te agregué

**Comment by Sauriorawr** [comment]
﻿Hola buenos días,  busco equipo voy poquito atrasado pero con las ganas de aprender todo se puede, dejo mi correo por si alguien todavía me puede meter en su ***@***.*** from my iPhoneOn 16 Nov 2022, at 6:30 a.m., David ***@***.***> wrote:﻿
Buen día, dejo mi correo por si todavia hay cupo!
***@***.***

**Comment by Hernanjas400** [comment]
Hola buenas estamos completos

Saludos

________________________________
De: Sauriorawr ***@***.***>
Enviado: miércoles, 16 de noviembre de 2022 18:03
Cc: Hernanjas400 ***@***.***>; Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Hola buenos dias!!!! Busco 5 integrantes con nivel inicial en html, css, y Javascript para la hackathon (#105)


﻿Hola buenos días,  busco equipo voy poquito atrasado pero con las ganas de aprender todo se puede, dejo mi correo por si alguien todavía me puede meter en su ***@***.*** from my iPhoneOn 16 Nov 2022, at 6:30 a.m., David ***@***.***> wrote:﻿
Buen día, dejo mi correo por si todavia hay cupo!
***@***.***

**Comment by Ayuya2001** [comment]
Esto ya no es por el curso cierto?, Es iniciativa de ustedes?
